site/app/templates/polls/AllPollsPageStudent.twig

```diff
@@ -1,73 +1,90 @@
 <div class="content">
     <h1> Polls </h1>
     <br/> <br/>
-    <h2> Today's Polls </h2>
-    <table class="table table-striped" style="width:100%;">
-            <col style="width: 33%">
-            <col style="width: 33%">
-            <col style="width: 33%">
-        <thead>
-            <tr>
-                <th scope="col">Name</th>
-                <th scope="col">Your Answer</th>
-                <th scope="col">View Poll</th>
-            </tr>
-        </thead>
-        <tbody>
-            {% for poll in todays_polls %}
+
+    <div class="dropdown_wrapper">
```

Comment:
Use dashes instead of underscores for CSS ids and classes see [here](https://submitty.org/developer/coding_style_guide/css)